client/scraper/general_metadata.py

```diff
@@ -1,21 +1,59 @@
-from HTMLParser import HTMLParser
-import json
-
 from metadata import Metadata
+from client.constants import JsonFields, MetadataFields
+from bs4 import BeautifulSoup
```

Comment:
put third party imports on a different line than inapp imports. Should look like this:
- native lib imports-
<new line>
- 3rd party lib imports 
<new line>
- our own imports